BUILDING_ON_WINDOWS.md

```diff
@@ -105,10 +105,10 @@ To produce a standalone package, you need to generate all required files using t
 
 To produce all supplement files for a standalone build, follow these steps (adjust paths as required):
 
-1.  Navigate to your build output directory with windows explorer, e.g. `C:\Users\example\src\build-chatterino-Desktop_Qt_5_15_2_MSVC2019_64bit-Release`
+1.  Navigate to your build output directory with Windows Explorer, e.g. `C:\Users\example\src\build-chatterino-Desktop_Qt_5_15_2_MSVC2019_64bit-Release`
 2.  Enter the `release` directory
 3.  Delete all files except the `chatterino.exe` file. You should be left with a directory only containing `chatterino.exe`.
-4.  Open a `cmd` window and execute:
+4.  Open a command prompt and execute:
```

Comment:
Please elaborate if you are okay with the way it says command prompt, or if it should directly say cmd instead